main/acle.md

```diff
@@ -9788,17 +9824,16 @@ Replacing `_hor` with `_ver` gives the associated vertical forms.
     __arm_streaming __arm_inout("za");
 ```
 
-#### FMOPA (non-widening)
+#### BFMOPA, FMOPA (non-widening)
 
 ``` c
+  // Variants are also available for:
+  //   _za16[_bf16]_m (only if __ARM_FEATURE_SME_B16B16 != 0)
+  //   _za16[_f16]_m (only if __ARM_FEATURE_SME_F16F16 != 0)
```

Comment:
I notice that the instructions under ARM_FEATURE_SME_F16F16 in the ACLE 
are only under FEATURE_SME_F16F16  in developer.arm(It looks like they do not require SME2), 

FMLA ZA.H[[<Wv>], [<offs>]{, VGx2}], { [<Zn1>].H-[<Zn2>].H }, [<Zm>].H[[<index>]]
if !IsFeatureImplemented(FEAT_SME_F16F16) then UNDEFINED;
I am not sure if the ACLE should follow that or not.

Reply:
There's a separate rule that:

> If FEAT_SME_F16F16 is implemented, then FEAT_SME2 is implemented.